packages/turbo-gen/src/utils/plop.ts

```diff
@@ -0,0 +1,222 @@
+import fs from "fs-extra";
+import { Project } from "@turbo/workspaces";
+import nodePlop, { NodePlopAPI, PlopCfg, PlopGenerator } from "node-plop";
+import path from "path";
+import inquirer from "inquirer";
+import { searchUp, getTurboConfigs, logger } from "@turbo/utils";
+
+// TODO: Support a TS config file
+const TURBO_GENERATOR_CONFIG = path.join("generators", "config.js");
+
+// support root plopfile so that users with existing configurations can use them immediately
+const DEFAULT_CONFIG_LOCATIONS = [
+  TURBO_GENERATOR_CONFIG,
+  "plopfile.js",
+  "plopfile.cjs",
+  "plopfile.mjs",
+];
```

Comment:
Should these additional config files be in the readme also?

Reply:
Yes will definitely add these to docs